testOPHD/MapOffsetTests.cpp

```diff
@@ -0,0 +1,105 @@
+#include "OPHD/Map/MapOffset.h"
+
+#include <gtest/gtest.h>
+
+
+TEST(MapOffset, EqualOperatorPass)
+{
+	MapOffset offset1;
+	offset1.xy = { 1, 2 };
+	offset1.z = 3;
+
+	MapOffset offset2;
+	offset2.xy = { 1, 2 };
+	offset2.z = 3;
+
+	EXPECT_TRUE(offset1 == offset2);
```

Comment:
This would be more terse with inline construction:
```cpp
EXPECT_TRUE(MapOffset{{1, 2}, 3} == MapOffset{{1, 2}, 3});
```

----

Additionally, this particular check might use `EXPECT_EQ` instead of `EXPECT_TRUE`. Not that it's much shorter or really changes readability.

```cpp
EXPECT_EQ(MapOffset{{1, 2}, 3}, MapOffset{{1, 2}, 3});
```

----

Similar for other tests too. Use inline construction. Using more specific expectations, such as `EXPECT_NE` (not equal).